packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -349,6 +349,7 @@ interface DesignPickerProps {
 	showActiveThemeBadge?: boolean;
 	isMultiFilterEnabled?: boolean;
 	isBigSkyEligible?: boolean;
+	isIntentCreateCourseGoalEnabled?: boolean;
```

Comment:
What do you think about passing the `[ 'education' ] = 'course'; directly as a prop instead of getting the feature flag outside and drilling down the value to the component? 

It will make the component prioritize configurable themes by category and reduce the coupling of the feature flag state.

Reply:
That's a good idea! I can do that when I'm removing the feature flag code.